src/curated-corpus/components/actions/EditCorpusItemAction/EditCorpusItemAction.test.tsx

```diff
@@ -56,7 +56,7 @@ describe('The EditCorpusItemAction', () => {
 
     // The most basic of integration tests - we simply send through
     // the corpus item without any actual edits.
-    userEvent.click(screen.getByText(/save/i));
+    userEvent.click(screen.getByText(/^save$/i));
```

Comment:
This was necessary because there are now multiple buttons containing 'save' in the text.